net/crowdsec-firewall-bouncer/files/crowdsec-firewall-bouncer.initd

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh /etc/rc.common
+# (C) 2021 Gerald Kerma
+
+START=99
+USE_PROCD=1
+NAME=crowdsec-firewall-bouncer
+PROG=/usr/bin/cs-firewall-bouncer
+CONFIG=/etc/crowdsec/bouncers/crowdsec-firewall-bouncer.yaml
+BACKEND=iptables
+VARCONFIGDIR=/var/etc/crowdsec/bouncers
+VARCONFIG=/var/etc/crowdsec/bouncers/crowdsec-firewall-bouncer.yaml
+FW_BACKEND="iptables"
+
+service_triggers() {
+	procd_add_reload_trigger crowdsec-firewall-bouncer
+}
+
+init_config() {
+	config_load crowdsec
+	config_get BACKEND crowdsec backend "${BACKEND}"
+
+	if [ "$BACKEND" = "auto" ]; then 
```

Comment:
```suggestion
	if [ "$BACKEND" != "iptables" ] && [ "$BACKEND" != "nftables" ]; then 
```

Reply:
removed, and now, backend is only in detection at service start...
thanks